staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,69 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, this directory will be skipped.
+- Adding `.import-restrictions` files does not add them to CI runs. They need to be explicitly added to a `packages` variable within `hack/verify-import-boss.sh`.
+
+  Here's an example code snippet of how it would look:
+  ```go
+  packages=("k8s.io/kubernetes/pkg/..." "k8s.io/kubernetes/cmd/..." "k8s.io/kubernetes/plugin/..." "k8s.io/kubernetes/test/e2e/framework/...")
+  ```
```

Comment:
i'd remove this example and leave it to the users to look at the file and find the packages variable.
it might change in the future so better not document it explicitly.

Reply:
Good point. Makes the document more resilient. Will update.